modules/multi-architecture-verifying-cluster-compatibility.adoc

```diff
@@ -41,6 +58,12 @@ You can then begin adding multi-arch compute nodes to your cluster.
 +
 [source,terminal]
 ----
-$ null
+{
+    "url": "https://access.redhat.com/errata/RHSA-2023:4671"
+}
 ----
-To migrate your cluster to one that supports multi-architecture compute machines, follow the procedure in "Migrating to a cluster with multi-architecture compute machines". 
\ No newline at end of file
+To migrate your cluster so that the cluster supports multi-architecture compute machines, follow the procedure in "Migrating to a cluster with multi-architecture compute machines".
```

Comment:
Do you want to add an Additional Resources under this module in the assembly so that users don't have to go searching for Migrating to a cluster with multi-architecture compute machines? 

I do see that it is x-reff'd in an IMPORTANT admonition in the following section, though, so up to you.